monitor/services/src/main/java/org/datacleaner/monitor/referencedata/ReferenceDataItem.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.monitor.referencedata;
+
+import java.io.Serializable;
+
+public class ReferenceDataItem implements Serializable {
+    public enum Type {
+        DICTIONARY,
+        SYNONYM_CATALOG,
+        STRING_PATTERN,
+    }
+
+    private String _name;
+    private Type _type;
+
+    public ReferenceDataItem() {
+    }
+
```

Comment:
That's weird. The default empty constructor should always be available when no other constructor is created.

Maybe it is some weird GWT parsing thing.